.github/workflows/temp-175-review-fixes.yml

```
@@ -0,0 +1,914 @@
+
+          def replace_regex(path, pattern, replacement):
+              p = Path(path)
+              text = p.read_text()
+              text, count = re.subn(pattern, replacement, text, count=1, flags=re.S)
+              if count != 1:
+                  raise SystemExit(f"expected one regex match in {path}, got {count}: {pattern}")
+              p.write_text(text)
+
+          Path("backend/internal/auth/api_keys_cache.go").write_text(r'''package auth
+
+import (
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)